samples/subsys/ipc/ipm_mcux/README.rst

```diff
@@ -0,0 +1,138 @@
+.. _ipm-mcux-sample:
+
+Sample mailbox application
+##########################
+
+Overview
+********
+
+This application will use mailbox to send messages from one processor core
```

Comment:
How about:

    The :ref:`lpcxpresso54114` board has two core processors (M4F and dual M0).
    This sample application uses a mailbox to send messages from one processor core
    to the other.

Reply:
Changed.